doc/api/errors.md

```diff
@@ -2445,6 +2445,19 @@ An invalid timestamp value was provided for a performance mark or measure.
 
 Invalid options were provided for a performance measure.
 
+<a id="ERR_PLATFORM_UNSUPPORTED_OPERATION"></a>
+
+### `ERR_PLATFORM_UNSUPPORTED_OPERATION`
```

Comment:
How does this differ from `ERR_FEATURE_UNAVAILABLE_ON_PLATFORM`?

Reply:
Simply I didn't find it. ;)
Removing it and using the existing one.